README.md

```diff
@@ -63,7 +63,7 @@ Try out Ballerina's development capabilities using the [Ballerina extension for
 >**Tip:** If you are unsure whether you have found a bug, search the existing issues in the GitHub repo and raise it in the [Ballerina Discord](https://discord.com/invite/wAJYFbMrG2) or [Stack Overflow](https://stackoverflow.com/questions/tagged/ballerina).
 
   - Language, Tooling, Ballerina library, Website: <a href="https://github.com/ballerina-platform/ballerina-lang/issues">ballerina-lang</a> repo
-  - Extended library: <a href="https://github.com/ballerina-platform/ballerina-extended-library/issues">ballerina-extended-library</a> repo
+  - Ballerina library: <a href="https://github.com/ballerina-platform/ballerina-extended-library/issues">ballerina-extended-library</a> repo
```

Comment:
Ballerina lib issues are here
https://github.com/ballerina-platform/ballerina-standard-library

Check with team are maintaining both repos or same

Reply:
Corrected.

Reply:
> Corrected. Need to wait for the repo rename and URL redirection.

Reply:
The repo is renamed and the URL is redirected now.